daemon/api_prompting.go

```diff
@@ -97,6 +99,92 @@ func getUserID(r *http.Request) (uint32, Response) {
 	return uint32(userIDInt), nil
 }
 
+func promptingNotRunningError() *apiError {
+	return &apiError{
+		Status:  500, // Internal error
+		Message: "AppArmor Prompting is not running",
+		Kind:    client.ErrorKindAppArmorPromptingNotRunning,
+	}
+}
+
+func promptingError(err error) *apiError {
+	apiErr := &apiError{
+		Message: err.Error(),
+	}
+	switch {
+	case errorIsOneOf(err, prompting.ErrPromptsClosed, prompting.ErrRulesClosed):
```

Comment:
not sure we need a dedicated helper:

```suggestion
	case errors.Is(err, prompting.ErrPromptsClosed) || errors.Is(err, prompting.ErrRulesClosed):
```